src/ui/screenshot_menu.css

```diff
@@ -14,25 +14,189 @@
  * limitations under the License.
  */
 
+* {
```

Comment:
The wildcard selector and .overlay change have adverse affects on the rest of the UI, we will need to move them into the specific sections of this .css file that they are needed, and remove the global wildcard selector and overlay changes